Comparison Engine/Comparison Engine/Form1.cs

```diff
@@ -98,6 +91,26 @@ private void buttonDrinksIsTop()
             buttonBottom.Text = "Bars";
         }
 
+        //loadBars and loadDrinks loads from List
+
+        private void loadBars()
+        {
+            clearButtonList();
+            foreach (Bar bar in bars)
+            {
+                createBarButton(bar);
+            }
+        }
+
+        private void loadDrinks()
+        {
+            clearButtonList();
+            foreach (Drink drink in drinks)
+            {
+                createDrinkButton(drink);
+            }
+        }
+
```

Comment:
These methods should probably take list of drinks/bars as arguments since the button list in main form should be updated every time search is completed.

Reply:
YOU GOT IT BOSS